src/main/java/com/puppycrawl/tools/checkstyle/checks/naming/ConstantNameCheck.java

```diff
@@ -64,7 +64,6 @@
  * <pre>
  * &lt;module name="ConstantName"/&gt;
  * </pre>
- *
```

Comment:
Please revert this change.

Reply:
This change was I did in my master repo them I branched, so this is the original one only.

Reply:
Resolved